src/SimpleGraphs/generators/staticgraphs.jl

```diff
@@ -763,3 +763,27 @@ function lollipop_graph(n1::T, n2::T) where {T <: Integer}
     add_edge!(g, n1, n1+1)
     return g
 end
+
+
+"""
+    friendship_graph(n)
+Create a [friendship graph](https://en.wikipedia.org/wiki/Friendship_graph)
+consisting of n copies of the cycle graph C3 with a common vertex`,
+the center vertex will always be 1, if n was <= 0 the function will return
+single node graph
+
+"""
+function friendship_graph(n::T) where {T <: Integer}
+    n <= 0 && return SimpleGraph(1)
+    g = SimpleGraph(2 * n + 1)
```

Comment:
```suggestion
    n <= 0 && return SimpleGraph{T}(1)
    g = SimpleGraph{T}(2 * n + 1)
```

Reply:
This is not a good idea since `2n + 1` can overflow `T`. It doesn't really matter if this function is type stable or not.